# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1490651826 -7200
#      Mon Mar 27 23:57:06 2017 +0200
# Branch stable
# Node ID 0f1e9743b0a96010610ca0110f74faaa0fb69d14
# Parent  71a749eeb7d3878411e791cfacba8791886c6dae
doc: describe optional install dependencies PortAudio and SUNDIALS (bug #50513).

* doc/interpreter/install.txi: Enrich the list of optional install dependecies
by PortAudio and SUNDIALS.

diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -282,16 +282,23 @@ interpreter.
 @item OpenGL
 API for portable 2-D and 3-D graphics (@url{http://www.opengl.org}).  An
 OpenGL implementation is required to provide Octave's OpenGL-based
 graphics functions.  Octave's OpenGL-based graphics functions usually
 outperform the gnuplot-based graphics functions because plot data can be
 rendered directly instead of sending data and commands to gnuplot for
 interpretation and rendering.
 
+@item PortAudio
+PortAudio (@url{http://www.portaudio.com/}) provides a very simple API
+for recording and/or playing sound using a simple callback function or
+a blocking read/write interface.  It is required for the audio processing
+functions @code{audioplayer}, @code{audiorecorder}, and
+@code{audiodevinfo}.
+
 @item Qhull
 Computational geometry library (@url{http://www.qhull.org}).  Qhull is
 required to provide the functions @code{convhull}, @code{convhulln},
 @code{delaunay}, @code{delaunayn}, @code{voronoi}, and @code{voronoin}.
 
 @item QRUPDATE
 QR factorization updating library
 (@url{http://sourceforge.net/projects/qrupdate}).  QRUPDATE is used to
@@ -310,16 +317,22 @@ It is a large framework, but the only co
 core, and network modules.
 
 @item SuiteSparse
 Sparse matrix factorization library
 (@url{http://www.cise.ufl.edu/research/sparse/SuiteSparse}).
 SuiteSparse is required to provide sparse matrix factorizations and
 solution of linear equations for sparse systems.
 
+@item SUNDIALS
+The SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers
+(@url{http://computation.llnl.gov/projects/sundials}) is required for
+the Ordinary Differential Equations (ODE) solvers @code{ode15i} and
+@code{ode15s}.
+
 @item zlib
 Data compression library (@url{http://zlib.net}).  The zlib library is
 required for Octave's @code{load} and @code{save} commands to handle
 compressed data, including @sc{matlab} v5 MAT files.
 @end table
 
 @node Running Configure and Make
 @section Running Configure and Make
